<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-13T13:35:01 GMT" name="Suite" finished-at="2025-03-13T13:35:26 GMT" duration-ms="24366">
    <groups>
    </groups>
    <test started-at="2025-03-13T13:35:01 GMT" name="Test2" finished-at="2025-03-13T13:35:05 GMT" duration-ms="3829">
      <class name="tests.Failintentional">
        <test-method signature="failedTest()[pri:0, instance:tests.Failintentional@36804139]" started-at="2025-03-13T13:35:01 GMT" name="failedTest" finished-at="2025-03-13T13:35:05 GMT" duration-ms="3430" status="FAIL">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:23)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
at tests.Failintentional.failedTest(Failintentional.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failedTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.Failintentional@36804139]" started-at="2025-03-13T13:35:05 GMT" name="tearDown" finished-at="2025-03-13T13:35:05 GMT" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.Failintentional -->
    </test> <!-- Test2 -->
    <test started-at="2025-03-13T13:35:01 GMT" name="Test" finished-at="2025-03-13T13:35:14 GMT" duration-ms="12788">
      <class name="tests.RegisterUser">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.RegisterUser@4d14b6c2]" started-at="2025-03-13T13:35:01 GMT" name="setUp" finished-at="2025-03-13T13:35:05 GMT" duration-ms="3494" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="userRegister()[pri:0, instance:tests.RegisterUser@4d14b6c2]" started-at="2025-03-13T13:35:05 GMT" name="userRegister" finished-at="2025-03-13T13:35:14 GMT" duration-ms="9169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userRegister -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.RegisterUser@4d14b6c2]" started-at="2025-03-13T13:35:14 GMT" name="tearDown" finished-at="2025-03-13T13:35:14 GMT" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.RegisterUser -->
    </test> <!-- Test -->
    <test started-at="2025-03-13T13:35:01 GMT" name="Test1" finished-at="2025-03-13T13:35:26 GMT" duration-ms="24366">
      <class name="tests.LoginUser">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:01 GMT" name="setUp" finished-at="2025-03-13T13:35:05 GMT" duration-ms="3631" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:05 GMT" name="Login" data-provider="loginData" finished-at="2025-03-13T13:35:10 GMT" duration-ms="4710" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[test77@9690.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ummar@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Success]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:10 GMT" name="tearDown" finished-at="2025-03-13T13:35:10 GMT" duration-ms="84" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:10 GMT" name="setUp" finished-at="2025-03-13T13:35:13 GMT" duration-ms="2958" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:13 GMT" name="Login" data-provider="loginData" finished-at="2025-03-13T13:35:26 GMT" duration-ms="12864" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[test77@9690.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Umar123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Failure]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginUser@7098b907]" started-at="2025-03-13T13:35:26 GMT" name="tearDown" finished-at="2025-03-13T13:35:26 GMT" duration-ms="109" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginUser -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
